cli/client.go

```diff
@@ -1975,6 +1976,32 @@ var clientCancelTransfer = &cli.Command{
 	},
 }
 
+var clientCancelRetrievalDeal = &cli.Command{
+	Name:  "cancel-retrieval-deal",
+	Usage: "Cancel a retrieval deal by DealID",
+	Flags: []cli.Flag{
+		&cli.Int64Flag{
+			Name:     "dealid",
+			Usage:    "specify retrieval deal by DealID",
+			Required: true,
+		},
+	},
+	Action: func(cctx *cli.Context) error {
+		api, closer, err := GetFullNodeAPI(cctx)
+		if err != nil {
+			return err
+		}
+		defer closer()
+		ctx := ReqContext(cctx)
+
+		if cctx.Int64("dealid") == 0 {
+			return errors.New("deal id cannot be 0")
```

Comment:
Are you sure we don't start that counter at 0?

Reply:
hmm turns out it *does* start at zero: https://github.com/filecoin-project/go-storedcounter/blob/master/storedcounter.go#L23
So we should only return an error if the deal id is < 0